ocs_ci/ocs/bucket_utils.py

```diff
@@ -980,7 +980,7 @@ def wait_for_pv_backingstore(backingstore_name, namespace=None):
 
     namespace = namespace or config.ENV_DATA["cluster_namespace"]
     sample = TimeoutSampler(
-        timeout=360,
+        timeout=1200,
```

Comment:
why do you need to increase the time here?

Reply:
This was added to test BS status taking more time than 360s or not.. Will remove this .